src/ftva_etl/metadata/marc.py

```diff
@@ -211,21 +213,29 @@ def _get_first_stripped(subfields: list[str]) -> str:
     name_of_part = _get_first_stripped(title_statement.get_subfields("p"))
     number_of_part = _get_first_stripped(title_statement.get_subfields("n"))
 
+    # TODO: Add logic to get `production_type` information from Filemaker record.
+
     # Handling the spec cases in reverse order,
     # to fail early and go from simplest to most complicated.
-    # CASE 5: No main title
+    # CASE 6: No main title
     if not main_title:
         # No main title (245 $a) is an error condition, indicating an invalid record.
         # Raise a ValueError to be handled by callers.
         raise ValueError(f"No 245 $a found in bib record {record_id}.")
 
-    # CASE 4: Main title, but no name of part or number of part
+    # CASE 5: Main title, but no name of part or number of part
     if main_title and not name_of_part and not number_of_part:
         titles["title"] = main_title
 
-    # CASE 3: Main title and number of part, but no name of part
-    if main_title and number_of_part and not name_of_part:
+    # CASE 4: Main title and number of part, but no name of part (for non-series)
+    if not is_series and main_title and number_of_part and not name_of_part:
+        titles["title"] = ". ".join([main_title, number_of_part])
+
+    # CASE 3: Main title and name of part, but no number of part (for series)
```

Comment:
The comment for case 3 does not match the code.  Code is right, comment is wrong.